src/coreclr/pal/src/arch/wasm/stubs.cpp

```diff
@@ -27,55 +27,56 @@ DBG_DebugBreak()
 extern "C" void
 RtlCaptureContext(OUT PCONTEXT ContextRecord)
 {
-    _ASSERT("RtlCaptureContext not implemented on wasm");
+    // we cannot implement this function for wasm because there is no way to capture the current execution context
```

Comment:
Would it be better to start fixing the callers instead? Where is this still used?

Reply:
This one is used in `DispatchManagedException` variants. It passes the context info in many places, so it would need lot of ifdefs. It seems better to just clear the context to possibly detect a problematic use later.

The rest is just fixing the asserts. We can try to clean up these (and others), it is out of scope of this PR, so I would rather keep this PR simple. Opened https://github.com/dotnet/runtime/issues/121839 to remember it